obelisk_ws/src/python/obelisk_leap_py/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>obelisk_leap_py</name>
+  <version>0.0.1</version>
+  <description>Python Obelisk ROS2 package for LEAP hand control.</description>
+  <maintainer email="alberthli@caltech.edu">alberthli</maintainer>
```

Comment:
you wrote this, so you should add your name/contact as maintainer (if there's a way to add multiple, maybe add me as well under you)